scripts/end_to_end_demo.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+sudo snap install microk8s --classic --channel=1.18/stable
+
+sudo usermod -a -G microk8s dazwilkin
```

Comment:
maybe `dazwilkin` should be $USER?

Reply:
Apologies! Yes, certainly not `dazwilkin`.

For DO this can be dropped as user interactions are as root too.

For GCP, it's non-trivial as the startup script runs as root but the user only be known when that user ssh's in.

Reply:
I think, because the subsequent commands all use `sudo microk8s.kubectl`, this line can be dropped.

Will redo both deployments to confirm the change, then update.

Reply:
maybe 

> Should [version check workflow](https://github.com/deislabs/akri/blob/170c790a4b96265b4defb9b6652875ca0b782d9c/.github/workflows/check-versioning.yml#L6) be updated to ignore this new scripts folder?

yes, i think so.

Reply:
can the startup script reference arguments for GCP?  it seems like you can almost, kinda, sort of do it: https://cloud.google.com/compute/docs/startupscript#custom

Reply:
assuming $USER during SSH would be a known id tied to your GCP account?

Reply:
Fixed: https://github.com/deislabs/akri/pull/56/commits/688c98a412a4078055ae42d57934b3f8bac5d742